packages/smart-accounts/src/light-account/accounts/base.ts

```diff
@@ -276,7 +276,7 @@ export async function toLightAccountBase<
               ...data,
               account: owner,
             })
-          : await signMessageAction({ account: owner, message });
+          : await signMessageAction({ account: owner, message: data });
```

Comment:
i think this was a bug, but this case is hard (or impossible?) to hit.

Reply:
I think this was impossible previously, but good fix.